sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileTable.scala

```diff
@@ -57,6 +61,12 @@ abstract class FileTable(
     val caseSensitive = sparkSession.sessionState.conf.caseSensitiveAnalysis
     SchemaUtils.checkColumnNameDuplication(dataSchema.fieldNames,
       "in the data schema", caseSensitive)
+    dataSchema.foreach { field =>
```

Comment:
@dongjoon-hyun The schema validation of read path is moved here.

Reply:
My bad. When I tested in `FileTableSuite` yesterday, I missed this part. Ya. Thanks, @gengliangwang and @HyukjinKwon .